static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -454,6 +454,8 @@ define([
             if (mobileAdSizes && mobileAdSizes.indexOf(adSizes.stickyMpu) > -1) {
                 stickyMpu($adSlot);
             }
+        } else if ($adSlot.hasClass('ad-slot--facebook')) {
+            $adSlot.addClass('ad-slot--fluid');
```

Comment:
Given the way that `then` operations are run a tick after the promise they listen to resolves, will there be a frame where the advert appears without its requisite styling? Perhaps we could just use `ad-slot--fluid` right from the start?